plane/source/docs/guided-mode.rst

```diff
@@ -4,12 +4,37 @@
 GUIDED Mode
 ===========
 
-The GUIDED mode is used when you want the aircraft to fly to a specific
-point on the map without setting up a mission. Most ground control
-stations support a "click to fly to" feature where you can click a point
-on the map and the aircraft will fly to that location then loiter.
+GUIDED mode is used when you want the aircraft to fly to a specific
+point on the map, or in specific direction (heading) without setting up a mission. 
 
 The other major use for GUIDED mode is in :ref:`geo-fencing <geofencing>`.
 When the geo-fence is breached the aircraft will enter GUIDED mode, and
 head to the predefined geo-fence return point, where it will loiter
 until the operator takes over.
+
+Location sub-mode
+-----------------
+Most ground control
+stations support a "click to fly to" feature where you can click a point
+on the map and the aircraft will fly to that location then loiter. If an 
+altitude is requested, the aircraft will try to achieve the target altitude as it flies,
+and will loiter down or up to the requested altitude once it arrives, if required. 
+
+This sub-mode can also be requested via MAVLink using the DO_REPOSITION command.
```

Comment:
isnt this the command GCS use when fly to is commanded?

Reply:
I think it is, I could mention that.